src/Symfony/Component/HttpKernel/Tests/Controller/ArgumentResolver/DateTimeValueResolverTest.php

```diff
@@ -20,7 +20,7 @@
 
 class DateTimeValueResolverTest extends TestCase
 {
-    private $defaultTimezone;
+    private readonly string $defaultTimezone;
```

Comment:
This class was introduced in 6.1, I took this PR as an opportunity to improve type definition and make data providers static.